Error: Cannot find module 'bluebird'

<https://github.com/bitfinexcom/bfx-hf-util/blob/master/lib/on_exit.js#L3>

I have found there is no `bluebird` on `package.json` in **bfx-hf-util** and it causes errors in `non-flatten` package management system like `pnpm`. 

Error message
```bash
> require('bitfinex-api-node');
Thrown:
{ Error: Cannot find module 'bluebird'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/getoken_code/node_modules/.verdaccio/bfx-hf-util/1.0.6/node_modules/bfx-hf-util/dist/lib/on_exit.js:3:15)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/getoken_code/node_modules/.verdaccio/bfx-hf-util/1.0.6/node_modules/bfx-hf-util/dist/lib/index.js:11:14)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18) code: 'MODULE_NOT_FOUND' }
```

Environment
* node 10.19.0
* pnpm 3.8.1

--- a/CHANGELOG
+++ b/CHANGELOG
@@ -1,3 +1,6 @@
+# 1.0.8
+- fix: add missing bluebird dep to manifest
+
 # 1.0.7
 - refactor: register on-exit listeners only if method is used
 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "bfx-hf-util",
-  "version": "1.0.7",
+  "version": "1.0.8",
   "description": "HF utilities",
   "main": "./dist/index.js",
   "directories": {
@@ -36,6 +36,7 @@
     "@babel/preset-es2015": "^7.0.0-beta.53",
     "@babel/preset-stage-0": "^7.0.0",
     "babel-eslint": "^7.2.2",
+    "bluebird": "^3.7.2",
     "copy": "^0.3.1",
     "create-index": "^2.3.0",
     "debug": "^4.1.1",

